modules/overview-of-accelerators.adoc

```diff
@@ -0,0 +1,28 @@
+:_module-type: CONCEPT
+
+[id='overview-of-accelerators_{context}']
+= Overview of accelerators
+
+[role='_abstract']
+If you work with large data sets, you can use accelerators to optimize the performance of your data science models in {productname-short}. Accelerators enable you to scale your work, reduce latency, and increase productivity. You can use accelerators in {productname-short} to assist your data scientists in the following tasks:
```

Comment:
Could avoid 'enable you' here by adjusting slightly to something like 'With accelerators, you can scale your work...'